bin/live/pycbc_live_combine_single_fits

```diff
@@ -48,8 +48,7 @@ if args.conservative_percentile < 50 or \
                  "otherwise it is either not a percentile, or not "
                  "conservative.")
 
-colours = ['r', (1.0, 0.6, 0), 'y', 'g', 'c', 'b', 'm',
-           (0.8, 0.25, 0), (0.25, 0.8, 0)]
+colours = ['r',(1.0,0.6,0),'y','g','c','b','m',(0.8,0.25,0),(0.25,0.8,0)]
```

Comment:
Adding more colours as we are looking at extending the number of bins

Reply:
is this adding more or just removing the spaces in the existing list?

Reply:
Note that you can actually use the colormaps to color lines. Maybe worth considering if you have many curves that are "indexed" by a continuous variable?

Reply:
That's a great idea - they are now indexed based on how far they are through the "rainbow_r" colormap (with an option to choose colormap)